lib/utils/dataflow/src/index.js

```diff
@@ -715,11 +735,11 @@ const mapper = (mapfn, opt) => {
       return {
         body: dbclient.undbify(doc)
       };
-    }));
+    });
 
   // Retrieve error docs
   if(opt.error && opt.error.get)
-    routes.get(opt.error.get, throttle(function *(req) {
```

Comment:
Wouldn't this cause a problem with too much incoming requests? Like out of threads/resources?

Do we rely on the batch calls to the pipeline and the throttling there?